src/modules/cards/cards.module.scss

```diff
@@ -51,18 +52,98 @@ $hover-color:        $neutral-2;
   }
 }
 
+$beneficiary-card-border-color:     $brand-3;
+$beneficiary-card-background-color: $neutral-8;
+$beneficiary-card-border-width:     $border-radius;
+$beneficiary-card-icon-size:        ru(1);
+$footer-background-color:           $neutral-6;
+
 .beneficiary {
   composes: card;
-}
 
-.col {
-  flex: 1;
-  border: 1px solid $color;
+  position: relative;
+  min-width: ru(11.33);
+  background-color: $beneficiary-card-background-color;
+  margin-bottom: ru(1);
+
+  .mod-percentage-card {
+    display: none;
+  }
+
+  &.collapsed {
```

Comment:
this should not be nested but instead composed from beneficiary card. `composes` will do the work of `&` behind the scenes.

This opens up the discussion of making individual Sass files for each module/component